taxi/views.py

```diff
@@ -34,6 +35,23 @@ class ManufacturerListView(LoginRequiredMixin, generic.ListView):
     paginate_by = 5
 
 
+class ManufacturerCreateView(LoginRequiredMixin, generic.CreateView):
+    model = Manufacturer
+    fields = "__all__"
+    success_url = reverse_lazy("taxi:manufacturer-list")
```

Comment:
Ensure that the form used in `ManufacturerCreateView` is using crispy forms as intended. This is a core requirement.